schunk_gripper_library/schunk_gripper_library/driver.py

```diff
@@ -295,11 +296,14 @@ def move_to_absolute_position(
         if not self.set_target_speed(velocity):
             return False
 
+        if is_absolute:
+            trigger_bit = 13
```

Comment:
I would always set the `trigger_bit = 13` for this PR. Else `move_to_position` with `is_absolute = False` raises an exception in line `306` because `trigger_bit` won't be declared.

Reply:
Done in b31be27ed25